src/node_crypto.cc

```diff
@@ -1154,6 +1163,12 @@ void SSLWrap<Base>::AddMethods(Environment* env, Handle<FunctionTemplate> t) {
   env->SetProtoMethod(t, "setNPNProtocols", SetNPNProtocols);
 #endif
 
+#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
+  NODE_SET_PROTOTYPE_METHOD(t, "getALPNNegotiatedProtocol",
```

Comment:
@shigeki I guess you can leave the #ifdef out.  I remember from the last review that it looked like it wouldn't compile with ALPN disabled but maybe I saw that wrong.